shared/test/otSdkDriver_test.js

```diff
@@ -157,6 +157,26 @@ describe("loop.OTSdkDriver", function() {
         sinon.match.instanceOf(HTMLDivElement),
         expectedConfig);
     });
+
+    it("should call onPublishComplete", function() {
+      sdk.initPublisher = sinon.stub().callsArg(2).returns(publisher);
+      driver._onPublishComplete = sinon.stub();
+
+      driver.setupStreamElements(new sharedActions.SetupStreamElements({
+        publisherConfig: publisherConfig
+      }));
+
+      sinon.assert.calledOnce(driver._onPublishComplete);
+    });
+
+    it("should dispatch failure when error in onPublishComplete", function() {
+      driver._onPublishComplete(true);
```

Comment:
I'd prefer this to be triggered in the same way as the current "onPublishComplete" does it, i.e. `sdk.initPublisher = sinon.stub().callsArg(2).returns(publisher);` then setupStreamElements.